### ISSUE TITLE
v1.2.0 No Longer Supports Python Version 3.8

### ISSUE THREAD

**Issue by TwoSails** [PythonistaGuild/Wavelink#139]
In the newest version ([v1.2.0](https://github.com/PythonistaGuild/Wavelink/releases/tag/v1.2.0)), the implementation of the YouTube Playlist searching has removed support for Python Version 3.8. The use of the method `.removeprefix()` is a python 3.9+ feature.

The line in question:
https://github.com/PythonistaGuild/Wavelink/blob/8a4fb12404a5e908bd7a22ecba1ace7dc950d73f/wavelink/tracks.py#L187

PEP 616:
String methods to remove prefixes and suffixes
https://www.python.org/dev/peps/pep-0616/ 

If this is intentional to remove Python 3.8 version, then please can the documentation be updated.
Otherwise, an implementation to get around this could be:
```py
def removePrefix(url, prefix):
    if string.startswith(prefix):
        return string[len(prefix):]
    else:
        return string
```

**Comment by EvieePy** [comment]
Thanks, been using 3.10 for most our new projects lately so this is a slight over look I will fix and release a new patch.

**Comment by TwoSails** [comment]
I have made an implementation in a fork if you want me to open a pull request?

**Comment by EvieePy** [comment]
Sure that's fine

**Comment by TwoSails** [comment]
Fixed in latest release v1.2.1 
Thank you!
